### PR TITLE
feat: 🎸 Update default to use chain v5.0.3

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Removes support for 4.x versions, as they were broken due to the images being removed from docker hub. Also updates the default REST API to v0.1.0

### Breaking Changes

- chain version 4 is no longer supported

### JIRA Link

[DA-428](https://polymesh.atlassian.net/browse/DA-428)

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/common/uis.ts [hunk 1]
```diff
@@ -69,42 +81,11 @@ function haveUIs(): boolean {
...
-interface S3Metadata {
-  Key: string[];
-  LastModified: string;
-}
-/**
- * Fetches the most recent UI tarball from the S3 bucket.
- * Uses the REST API since the official S3 client requires credentials
- * @returns The S3 filename of the most recent UI for the given version
- */
-async function fetchFileName(version: string): Promise<Record<string, unknown> | void> {
```
Comment (polymath-eric) [inline src/common/uis.ts:81]: In hindsight, S3 isn't great for this kind of stuff.

**Comment by adamdossa** [comment]
Not directly related to this PR, but https://github.com/PolymeshAssociation/polymesh-local/blob/master/scripts/build-uis.ts#L21 can be updated to the `PolymeshAssociation` repo https://github.com/PolymeshAssociation/governance-web.

**Comment by polymath-eric** [comment]
Thanks @adamdossa . I updated that reference, removed the deprecated upload script, and added a blurb in the README to document the process a bit better.

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**